### DISCUSSION TITLE
Duda respecto a archivos a subir al repositorio de la tarea

### DISCUSSION THREAD

**Discussion by BenjaCarrill0** [IIC2133-PUC/2024-1#55] [Tarea 0] [answered]
Hola! Me surgió la duda acerca de qué archivos hay que subir al repositorio, se que se debe subir tanto el informe como el código, pero ¿se debe subir también el archivo compilado y también un README?

**Comment by PabloNavarroY** [comment, marked as answer]
Vamos a compilar tu código fuente localmente, por lo que no es necesario que subas el ejecutable. No es necesario que ocupes el README.
